compiler/quilt/tools/store.py

```diff
@@ -205,47 +206,56 @@ def iterpackages(self):
+        for team in os.listdir(pkgdir):
+            for user in os.listdir(self.team_path(team)):
+                for pkg in os.listdir(self.user_path(team, user)):
+                    pkgpath = self.package_path(team, user, pkg)
+                    for hsh in os.listdir(os.path.join(pkgpath, Package.CONTENTS_DIR)):
+                        yield Package(self, user, pkg, pkgpath, pkghash=hsh)
 
     def ls_packages(self):
         """
         List packages in this store.
         """
         packages = []
         pkgdir = os.path.join(self._path, self.PKG_DIR)
-        for user in os.listdir(pkgdir):
-            for pkg in os.listdir(os.path.join(pkgdir, user)):
-                pkgpath = os.path.join(pkgdir, user, pkg)           
-                pkgmap = {h : [] for h in os.listdir(os.path.join(pkgpath, Package.CONTENTS_DIR))}
-                for tag in os.listdir(os.path.join(pkgpath, Package.TAGS_DIR)):
-                    with open(os.path.join(pkgpath, Package.TAGS_DIR, tag), 'r') as tagfile:
-                        pkghash = tagfile.read()
-                        pkgmap[pkghash].append(tag)
-                for pkghash, tags in pkgmap.items():
-                    fullpkg = "{owner}/{pkg}".format(owner=user, pkg=pkg)
-                    # Add an empty string tag for untagged hashes
-                    displaytags = tags if tags else [""]
-                    # Display a separate full line per tag like Docker
-                    for tag in displaytags:
-                        packages.append((fullpkg, str(tag), pkghash))
+        for team in os.listdir(pkgdir):
+            for user in os.listdir(self.team_path(team)):
+                for pkg in os.listdir(self.user_path(team, user)):
+                    pkgpath = self.package_path(team, user, pkg)
+                    pkgmap = {h : [] for h in os.listdir(os.path.join(pkgpath, Package.CONTENTS_DIR))}
+                    for tag in os.listdir(os.path.join(pkgpath, Package.TAGS_DIR)):
+                        with open(os.path.join(pkgpath, Package.TAGS_DIR, tag), 'r') as tagfile:
+                            pkghash = tagfile.read()
+                            pkgmap[pkghash].append(tag)
+                    for pkghash, tags in pkgmap.items():
+                        fullpkg = "{owner}/{pkg}".format(owner=user, pkg=pkg)
+                        # Add an empty string tag for untagged hashes
+                        displaytags = tags if tags else [""]
+                        # Display a separate full line per tag like Docker
+                        for tag in displaytags:
+                            packages.append((fullpkg, str(tag), pkghash))
                         
         return packages
 
-    def user_path(self, user):
+    def team_path(self, team):
+        """
+        Returns the path to directory with the team's users' package repositories.
+        """
+        teamdir = team if team else DEFAULT_TEAM
```

Comment:
seems like this is just a default param i.e. team_path(self, team=DEFAULT_TEAM) ?

Reply:
will fix